pages/Radiation.md

```diff
@@ -0,0 +1,18 @@
+Radiation is an effect of radioactive items in Slimefun.
+
+More specifically, it is a deadly combination of status effects you receive when carrying such items: Wither III, Blindness IV, Confusion IV, Weakness III, Slowness II, and Mining Fatigue II, for a duration of 20 seconds.
+
+For this reason, it is highly recommended you only carry them for a short amount of time, or, better yet, wear a full [hazmat suit](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Hazmat-Suit) while handling them.
+
```

Comment:
I think you should add a sentence there.
It's not immediately clear that the full Hazmat Suit will make you immune to radiation.

Reply:
Right on, I hope this phrasing makes it more clear